Comparison of signed and unsigned int in JitterBuffer.cpp

I've might accidentally dropped this from the clazy fixes PR yesterday...

```
../src/JitterBuffer.cpp:247:21: warning: comparison of integers of different signs: 'int' and 'unsigned int' [-Wsign-compare]
    if (std::abs(d) > mBroadcastLatency / 2) {
        ~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/JitterBuffer.cpp
+++ b/src/JitterBuffer.cpp
@@ -244,7 +244,7 @@ void JitterBuffer::readBroadcastSlot(int8_t* ptrToReadSlot)
     }
     // latency correction
     int32_t d = mReadPosition - mBroadcastLatency - mBroadcastPosition - len;
-    if (std::abs(d) > mBroadcastLatency / 2) {
+    if (static_cast<uint32_t>(std::abs(d)) > mBroadcastLatency / 2) {
         mBroadcastPosition     = mReadPosition - mBroadcastLatency - len;
         mBroadcastPositionCorr = 0.0;
         mBroadcastSkew += d / mMinStepSize;

